### PR TITLE
Update security-considerations.md

### PR DESCRIPTION
**Pull request by d-beamon**
## What This PR Changes
- I have suggested changes to make some of the terms more clear and consistent with the definitions on the Cloud subsite. I have also suggested that we extract the boards security considerations from the Applications and Tools, so that it has its own section.

## Contribution Guidelines
- [X] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md [hunk 1]
```diff
@@ -121,26 +121,31 @@ If it is determined that a breach has occurred, system(s) or system component(s)
 
 PII data will only be stored as long as Arduino legitimately needs it and in accordance with Arduino’s Privacy Policy.
 
-## Security Considerations for Specific Arduino Cloud Components
+## Security Considerations for Specific Arduino Cloud applications and tools
```
Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:124]:
```suggestion
## Arduino Cloud Applications and Tools
```

Comment (jhansson-ard) [reply content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:124]: I would shorten this heading, and seeing as we already are speaking about "security considerations" maybe we won't need it in the heading?

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md [hunk 2]
```diff
@@ -121,26 +121,31 @@ If it is determined that a breach has occurred, system(s) or system component(s)
...
-The Arduino Cloud Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. Wi-Fi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
-In order to communicate with the Arduino Cloud Editor, web browsers should be able to communicate over the internet to port 80, 443 and 8080 towards Arduino’s subdomains \*.arduino.cc.
+The Arduino Web Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. Wi-Fi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
+In order to communicate with the Arduino Web Editor, web browsers should be able to communicate over the internet to port 80, 443 and 8080 towards Arduino’s subdomains \*.arduino.cc.
 
 ### Arduino IoT Cloud
 
 The Arduino IoT Cloud collects MQTT events through a data pipeline which is accessible only by the owner of the data and access authorization is enforced via our backend API.
 
-### Arduino Boards Security
```
Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:141]: @d-beamon Why was this heading removed?

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md (hunk 1 above)
Comment (d-beamon) [reply content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:124]: I'm fine with the suggestion

**Comment by jhansson-ard** [comment]
@d-beamon please sign the CLA as well 🙏🏼

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md [hunk 3]
```diff
@@ -121,26 +121,31 @@ If it is determined that a breach has occurred, system(s) or system component(s)
...
 
-The Arduino Cloud Editor allows customers to create sketches they can subsequently deploy on boards. Each customer has a separate area where their sketches are stored so that no other customer has access to them. Sketches can be made publicly available to everyone, in which case access is granted by whomever possesses the link to access them.
-
-The Arduino Cloud Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. Wi-Fi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
-In order to communicate with the Arduino Cloud Editor, web browsers should be able to communicate over the internet to port 80, 443 and 8080 towards Arduino’s subdomains \*.arduino.cc.
+The Arduino Web Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. Wi-Fi credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
```
Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:130]:
```suggestion
The Arduino Web Editor also provides a secret tab that can be used to securely store sensitive information needed by the device (e.g. Wi-Fi® credentials, API credentials). Secrets in this tab are encrypted with the AES-256 algorithm to safely store them in the Cloud.
```

File: content/arduino-cloud/02.features/00.security-considerations/security-considerations.md [hunk 4]
```diff
@@ -121,26 +121,31 @@ If it is determined that a breach has occurred, system(s) or system component(s)
...
+
+For usage with certain browsers, specifically with Safari on macOS, the Arduino Create Agent needs to run the web server using a TLS connection. A local Certificate Authority (CA) is generated and used to sign a certificate for the localhost domain. The private key of the CA is erased once the certificate for localhost is signed, while the public certificate of the CA is installed in the local system. The web server of the Arduino Create Agent will then run with a TLS connection. This is done to obey the mixed content specification implemented in Safari which is enforced also for resources loaded from localhost.
```
Comment (jhansson-ard) [inline content/arduino-cloud/02.features/00.security-considerations/security-considerations.md:145]:
```suggestion
For usage with certain browsers, specifically with Safari on macOS, the Arduino Create Agent needs to run the web server using a TLS connection. A local Certificate Authority (CA) is generated and used to sign a certificate for the localhost domain. The private key of the CA is erased once the certificate for localhost is signed, while the public certificate of the CA is installed in the local system. The web server of the Arduino Create Agent will then run with a TLS connection. This is done to obey the mixed content specification implemented in Safari which is enforced also for resources loaded from localhost.

```

**Review by jhansson-ard — APPROVED** [review]
Nice! Approved 💪🏼

**Pull request merged**